### PR TITLE
Buxfix/refresh expansion

### PR DESCRIPTION
**Pull request by codenjoyme**

### CODE REVIEW

File: CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java [hunk 1]
```diff
@@ -92,8 +101,29 @@ public void update(Object data) {
...
         scores.put(SCORE, score);
+        scores.put(DETAILS, details());
         return scores;
     }
+
+    private String details() {
+        List<Object> list = rounds().toList();
+        int count = list.size();
+        int sum = list.stream()
+                .map(it -> (Integer) it)
```
Comment (rkulyniak) [inline CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java:112]: this line is useless
you can map to int immediately
`list.stream().mapToInt(obj -> (Integer) obj).sum()`

File: CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java [hunk 2]
```diff
@@ -92,8 +101,29 @@ public void update(Object data) {
...
+                .mapToInt(it -> it).sum();
+        double average = (count == 0) ? 0 : (100 * sum) / (count * 4.0);
```
Comment (rkulyniak) [inline CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java:114]: Have you considered computing an average value by using stream API?
`list.stream().mapToInt(it -> (Integer) it).average()`

**Review by rkulyniak — APPROVED** [review]
minor things to improve

File: CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java (hunk 2 above)
Comment (codenjoyme) [reply CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java:114]: double average = list.stream()
                .mapToInt(it -> it)
                .average().orElse(0.0) * 25.0;

File: CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java (hunk 1 above)
Comment (codenjoyme) [reply CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java:112]: Done

File: CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java (hunk 2 above)
Comment (codenjoyme) [reply CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java:114]: Done 
image

**Pull request merged**